### ISSUE TITLE
Kiwi image editor: Redirect directly to the kiwi editor when coming from template page

### ISSUE THREAD

**Issue by ChrisBr** [openSUSE/open-build-service#3932]
Issue/Feature description
====================
When branching from the template page, user will get redirected to package#show and then needs to click on the "Edit kiwi" link. This is not a nice UX and most users will probably have problems to find the "Edit kiwi" link.

Expected result
============
Directly redirect to the kiwi editor.

**Comment by adrianschroeter** [comment]
Good idea, but we may rethink this editor page then and also show more details, like build state and offering downloads. The user would not need to leave the simpler "Studio Express" interface for the covered/simpler tasks.

**Comment by adrianschroeter** [comment]
https://trello.com/c/fog41avq/557-improve-studio-express-experience
